.github/workflows/ci-branches.yaml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
     - '*'
     - '!gh-pages'
+    - '!ui-review*'
```

Comment:
I think we should also keep this branch pattern in `ci-prs.yaml`, `oursky.yaml`, and `custom-build.yaml`

Reply:
Added a commit to skip these workflows